src/libs/ErrorUtils.js

```diff
@@ -35,7 +36,23 @@ function getAuthenticateErrorMessage(response) {
     }
 }
 
+/**
+ * @param {Object} onyxData
+ * @param {Array} onyxData.errors
```

Comment:
I think the `onyxData.errors` is an `{Object}`, not an `{Array}`

<img width="378" alt="image" src="https://user-images.githubusercontent.com/87341702/186114303-3258e55f-018a-4fcc-a759-5ffc9f65335f.png">